tests/local.mk

```diff
@@ -41,39 +41,3 @@ install-crds: kind-setup ## Install CRDs of providers in kind cluster.
 		-f "https://raw.githubusercontent.com/vshn/provider-exoscale/$(provider_exoscale_version)/package/crds/exoscale.crossplane.io_buckets.yaml" \
 		-f "https://raw.githubusercontent.com/vshn/provider-exoscale/$(provider_exoscale_version)/package/crds/exoscale.crossplane.io_iamkeys.yaml" \
 		-f "https://raw.githubusercontent.com/crossplane-contrib/provider-kubernetes/$(provider_kubernetes_version)/package/crds/kubernetes.crossplane.io_objects.yaml" \
-
-.exoscale-composition:
-	$(MAKE) .prepare-integration-tests -e instance=exoscale
-
-.cloudscale-composition:
-	$(MAKE) .prepare-integration-tests -e instance=cloudscale
-
-.PHONY: .prepare-integration-tests
-.prepare-integration-tests:
-	rm -rf .cache compiled dependencies vendor
-	cp $(golden_dir)/composite-objectstorage-$(instance)/appcat/appcat/composites.yaml tests/kuttl/$(instance)-status-test/00-install-$(instance)-composite.yaml
```

Comment:
I think we can pretty much delete the whole file. I don't see another reason to install a local cluster.